conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/config/LorreAppConfig.scala

```diff
@@ -132,26 +134,37 @@ object LorreAppConfig {
     def unapply(s: String): Option[Int] = util.Try(s.toInt).filter(_ > 0).toOption
   }
 
-  private[config] object Loaders {
+  private[config] object Loaders extends PlatformConfigurationHint {
 
     /*** Reads a specific platform configuration based on given 'platform' and 'network' */
     def loadPlatformConfiguration(
         platform: String,
         network: String
-    ): Either[ConfigReaderFailures, PlatformConfiguration] = {
-      @silent("local method hint in method loadPlatformConfiguration is never used")
-      implicit def hint[T]: ProductHint[T] = ProductHint[T](ConfigFieldMapping(CamelCase, CamelCase))
+    ): Either[ConfigReaderFailures, PlatformConfiguration] =
       // Note that Lorre process allows to run only one integration per process.
       // Configuration file can contain more than one, thus required parameters.
-      BlockchainPlatform.fromString(platform) match {
-        case Tezos =>
-          for {
-            node <- loadConfig[TezosNodeConfiguration](namespace = s"platforms.$platform.$network.node")
-            tns <- loadConfig[Option[TNSContractConfiguration]](namespace = s"tns.$network")
-          } yield TezosConfiguration(network, node, tns)
-        case UnknownPlatform(_) => Right(UnknownPlatformConfiguration(network))
-      }
-    }
+      for {
+        config <- loadConfig[PlatformsConfiguration].flatMap {
+          _.platforms.find(x => x.platform.name == platform && x.network == network) match {
+            case Some(value) if value.enabled => Right(value)
```

Comment:
As before, I might prefer (personal tastes) that a specific name here would be used, to convey the meaning to the reader, without having to figure out the `value` type.

I tend to go in that direction because my goal is to make the code easy to understand for the reader, which is already a hard task! 😀